tests/unit/test_utils.py

```diff
@@ -321,18 +322,13 @@ def test_create_process_on_stdin_method_call(self):
                 env={'LESS': 'ABC'}
             )
 
-    def test_create_process_on_process_method_call(self):
```

Comment:
The diff is confusing me here, is this test no longer needed?

Reply:
yes because just a `communicate` call doesn't create pager object any more